primitives/npos-elections/src/tests.rs

```diff
@@ -72,6 +72,46 @@ fn float_phragmen_poc_works() {
 	);
 }
 
+#[test]
+fn phragmen_core_test_without_edges() {
+	let candidates = vec![1, 2, 3];
+	let voters = vec![
+		(10, 10, vec![]),
```

Comment:
we could also test for non candidate vote here
```suggestion
		(10, 10, vec![4, 5, 6]),
```